MAUI/Expander/Appearance.md

```diff
@@ -0,0 +1,149 @@
+---
+layout: post
+title: Appearance in .NET MAUI Expander control | Syncfusion
+description: Learn here all about Appearance support in Syncfusion .NET MAUI Expander (SfExpander) control and more.
+platform: MAUI
+control: SfExpander
+documentation: ug
+---
+
+# Appearance in .NET MAUI Expander (SfExpander)
+
+The Expander allows to customize appearance of the Icon, and provides various functionalities to the users.
+
+## Header icon position 
+
+The `SfExpander` allows you to customize the position of the header icon by using the `HeaderIconPosition` property. By default, the header icon position is `Start`. 
```

Comment:
The `SfExpander` allows you to customize the position of the header icon by using the `HeaderIconPosition` property. By default, the header icon position is `Start.`

Reply:
Addressed the concerns provided.